### PR TITLE
Replaced 'swagger.yaml' with openapi.yaml. Refer to b/32670418.

### PR DESCRIPTION
**Pull request by jasminejaksic**

### CODE REVIEW

**Comment by elharo** [comment]
What is the situation on the server? What does it do with a file named swagger.yaml? I'm fine with adding openapi.yaml to to the whitelist, but if clients are still uploading and the server is still processing swagger.yaml we should probably keep that too.

**Comment by jasminejaksic** [comment]
The file name should not matter on the server side. 
All the external doc references will also be named (refer to CL/140226260).

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 public class CloudSdkAppEngineFlexibleStaging implements AppEngineFlexibleStaging {
   protected static final Set<String> APP_ENGINE_CONFIG_FILES_WHITELIST = ImmutableSet.of("app.yaml",
       "cron.yaml", "queue.yaml", "dispatch.yaml", "index.yaml", "dos.yaml", "swagger.json",
-      "swagger.yaml");
+      "openapi.yaml");
```
Comment (elharo) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java]: why did you change swagger.yaml --> openapi.yaml and not change swagger.json --> openapi.json?

Comment (jasminejaksic) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java]: All the references will be changed sooner or later. For now I have edited the comments and replaced swagger.yaml.

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java]: I'm not sure I understand the explanation. Preferably would like to see everything changed that needs to be changed.

**Comment by loosebazooka** [comment]
I'm going to append to this pr to change the json file as well.

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java (hunk 1 above)
Comment (elharo) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineFlexibleStaging.java]: This is not an example. This is production code. It needs to support the file names that actually appear in the wild.

**Comment by patflynn** [comment]
presumably somewhere in gcloud there's a switch that tells it what to do
when it's deploying 'swagger.yam' or 'openapi.yaml'.  It seems like it
would have to be filename specific.
